### ISSUE TITLE
Allow "structure-folders/-documents" in FURL-generation

### ISSUE THREAD

**Issue by JensWolff** [modx-ccc-2015/whishlist#21]
Sometimes it is necessary to use modx-resources as "structure-folders" for example for navigation-generation.
As example, we want to build a custom footer-navigation, which should be reordered via the MODX Resource-Tree. So you create a new resource "Footer-Navigation" and put all the Ressources that should be displayed in that navigation as childs to display them f.ex. with wayfinder.
That works - but the friendly urls that gets generated are very ugly:
/footer-navigation/impress
or
/footer-navigation/contact

You could fix that by setting manually a fixed URL and clear it for all these ressources, that are childs of "footer-navigation", but this is not the best way to do it, because you always have to remember to manually fix them.

A better way would be, if you can check a checkbox (like "do not show in menu") like "hide in furl-generation". If this checkbox gets checked, the MODX FURL-generator ignores "footer-navigation" completely, so you get the following FURLS automaticly generated:

/impress
/contact

if you uncheck it, it will becomes again:
/footer-navigation/impress
/footer-navigation/contact

**Comment by wuuti** [comment]
Is this not covered by the "use_alias_path" setting? If this is set to false (default), then the structure folder won't be part of the url?

**Comment by JensWolff** [comment]
@inreti Yes, if you don't need any subfolders, then thats the right way to do it. But I generally want MODX to use_alias_path to create FURLS like news/news-article and so on. But I have some "folders" (=resources) that are just used for construction and sorting, but these should not be visible in the generated FURLs.

**Comment by rtripault** [comment]
Just to note there is discussion (and PoC) about this on https://github.com/modxcms/revolution/issues/11153

**Comment by gpsietzema** [comment]
+1

Also this PR: https://github.com/modxcms/revolution/pull/11167
